cmd/basin/publication.go

```diff
@@ -646,17 +646,29 @@ func inspectTable(ctx context.Context, tx pgx.Tx, rel string) ([]app.Column, err
 					ccu.column_name
 			FROM information_schema.table_constraints tc
 			JOIN information_schema.constraint_column_usage AS ccu USING (CONSTRAINT_SCHEMA, CONSTRAINT_NAME)
-			WHERE constraint_type = 'PRIMARY KEY' )
+			WHERE constraint_type = 'PRIMARY KEY' ),
+			array_type_info AS
+				(SELECT c.table_name,
+						c.column_name,
+						pg_catalog.format_type(t.oid, NULL) AS full_data_type
+				FROM information_schema.columns AS c
+				JOIN pg_catalog.pg_type AS t ON c.udt_name = t.typname
```

Comment:
Enhances the inspect query to get metadata for columns of type `ARRAY`.  


<img width="689" alt="Screenshot 2023-11-29 at 01 31 04" src="https://github.com/tablelandnetwork/basin-cli/assets/5305984/6e7f1dd2-35bf-4440-845e-95fdd65eaba3">